Packs/Armorblox/Integrations/Armorblox/Armorblox.yml

```diff
@@ -0,0 +1,85 @@
+category: Email Gateway
+commonfields:
+  id: Armorblox
+  version: -1
+configuration:
+- defaultvalue: outbount-integration-tenant
```

Comment:
Can you remove this value? Also remove any other occurrences of this value in this PR.